### ISSUE TITLE
Error whit CometChat in WebView Android

### ISSUE THREAD

**Issue by andreacona96** [cometchat-pro-samples/android-java-chat-app#70]
**Describe the bug**
Hi,
we're encountering an issue with CometChat on the Android platform.
We're attempting to load CometChat within a WebView, but we're facing an error. The attached file displays a "BLOCKED_BY_ORB" error.

The WebView has been configured with the following settings:

WebSettings webSettings = webView.getSettings();
webSettings.setJavaScriptEnabled(true); // Enables JavaScript
webSettings.setBuiltInZoomControls(false); // Disables built-in zoom controls
webSettings.setCacheMode(WebSettings.LOAD_NO_CACHE); // Sets the cache mode to not use cache
webSettings.setDomStorageEnabled(true); // Enables DOM storage
webSettings.setSupportZoom(false); // Disables zoom support

In debug mode we see all API calls go successfully as shown in the screenshot below.
The url uploaded by the Webview is https:///www.u4fit.com/it/chat/comet.

Could you assist us in resolving this issue, please?

**Screenshots**

<img width="276" alt="Screenshot 2024-04-19 alle 16 22 28" src="https://github.com/cometchat/cometchat-sample-app-android-java/assets/58525601/566edc26-d0a7-4e0f-8bff-9e4841c54fad">

<img width="506" alt="Screenshot 2024-04-19 alle 16 33 54" src="https://github.com/cometchat/cometchat-sample-app-android-java/assets/58525601/090e8cfa-bbab-4bf3-bec6-e70dccfcf471">

<img width="999" alt="Screenshot 2024-05-10 alle 16 51 44" src="https://github.com/cometchat/cometchat-sample-app-android-java/assets/58525601/ee272f84-5793-4bbd-8b34-600dba5afeb0">

**Comment by andreacona96** [comment]
Hi,
we use the on-premise version of CometChat for web chat. To connect from Android devices, we utilize a WebView. The details of the version used are shown in the attached screenshot.

<img width="565" alt="Screenshot 2024-05-10 alle 18 21 33" src="https://github.com/cometchat/cometchat-sample-app-android-java/assets/58525601/c3fd8c1a-074f-478a-9554-433fc6a565cd">

**Comment by andreacona96** [comment]
For the benefit of others who may have the same problem in the future, I am communicating that we have solved it ourselves. 
The problem was due to an update of some Google endpoints, specifically this URL https://www.google.com/uds/?file=elements&v=1&packages=transliteration was changed to this other one https://www.google.com/jsapi/?file=elements&v=1&packages=transliteration.

We've update all the references and dependencies related to the old URL to restore the service. I remember that in our case the issue was only affecting the Android platform.
